wasm/examples/parcel_example/index.js

```diff
@@ -147,14 +135,6 @@ async function renderRiveAnimation({ rive, num, hasRandomSizes }) {
     }
     renderer.flush();
 
-    // Update frame time
-    const after = performance.now();
-    durations.push(after - before);
-    // Use average of all recent durations
-    document.getElementById("framems-value").innerText = (
```

Comment:
Due to the various operations that may or may not be deferred, depending on a variety of reasons, I usually find single frame time measurements to be unhelpful and even misleading. I just took it out of here, but can add it back to the global Rive fps counter if it's something we want.